Model/Cron.php

```diff
@@ -2121,24 +2091,24 @@ protected function finalizeOrder($manualReviewComment = true, $createInvoice = f
 
         if ($fullAmountFinalized) {
             $comment = "Adyen Payment Successfully completed";
-
-            // If manual review is true AND manual review status is set
-            if ($manualReviewComment && $this->requireFraudManualReview) {
-                // check if different status is selected
-                $fraudManualReviewStatus = $this->getFraudManualReviewStatus();
-                if ($fraudManualReviewStatus != "") {
-                    $status = $fraudManualReviewStatus;
-                    $comment = "Adyen Payment is in Manual Review check the Adyen platform";
+            // If a status is set, add comment, set status and update the state based on the status
+            // Else add comment and if on auto capture, set state to payment review (replicate previous functionality)
+            // TODO: Check if state should be set to PAYMENT_REVIEW
```

Comment:
How should we check this?

Reply:
We checked this by comparing the previous functioning flows and ensuring that with this PR, those flows end up in the same state. Comment removed in https://github.com/Adyen/adyen-magento2/commit/8da51746ce736d1804db34d8d426325b11d56448